lib/membrane_element_file/source.ex

```diff
@@ -5,7 +5,8 @@ defmodule Membrane.Element.File.Source do
   """
 
   use Membrane.Source
-  alias Membrane.{Buffer, Event}
+  alias Membrane.Buffer
+  alias Membrane.Core.Events
```

Comment:
these events became private, see 0.6.0 release notes

Reply:
Oh, that's right. Fixed :+1: